app/views/layouts/application.html.haml

```diff
@@ -17,13 +17,12 @@
       = yield :head
 
   %body{'data-environment' => Rails.env}
+    = link_to "Skip to content", "#main", class: "skip-to-content-link"
```

Comment:
Since this only would be required on the room view since that is where the filters we are trying to skip are present, moving the link to the top of the app/views/rooms/index.html.erb.

This would require tabbing through the header items on each page still but that maybe a normal flow from a usability perspective. Maria L. would be able to clarify that.

Reply:
fixed in current rendition of pull request